fog/distribution/src/main.rs

```diff
@@ -125,11 +151,13 @@ fn main() {
 
     let ledger_db = LedgerDB::open(ledger_dir.path()).expect("Could not open ledger_db");
 
+    let mut block_version = ledger_db.get_latest_block().unwrap().version;
+    if let Some(config_block_version) = config.block_version.as_ref() {
+        block_version = *config_block_version;
+    }
```

Comment:
Nit: This should not require block_version to be mut - unwrap_or_else can be used to fallback to the latest block version.